policyengine_us/parameters/gov/states/oh/tax/income/exemptions/personal/amount.yaml

```
@@ -0,0 +1,27 @@
+description: Ohio provides this personal exemption amount, based on the filers modified adjusted gross income. 
+brackets:
+  - threshold:
+      2021-01-01: 0
+    amount:
+      2021-01-01: 2_400
+  - threshold: 
+      2021-01-01: 40_001
```

Same as below. Use `right=true` in the calc call instead
```suggestion
      2021-01-01: 40_000
```